modules/enabling-etcd-encryption.adoc

```diff
@@ -30,8 +30,9 @@ $ oc edit apiserver
 ----
 spec:
   encryption:
-    type: aescbc
+    type: aescbc <1>
 ----
+<1> The `aescbc` type means that AES-CBC with PKCS#7 padding and a 32 byte key is used to perform the encryption.
```

Comment:
I think that we need to point out that this is a FIPS approved cryptography package per https://github.com/openshift/openshift-docs/pull/18428#discussion_r356785590, but I will probably add it to that PR after you merge your PR.